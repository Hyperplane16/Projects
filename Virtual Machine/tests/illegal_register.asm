FUNC LABEL 0
    MOV REG 0 VAL 25
    MOV REG 5 VAL 30
    ADD REG 0 REG 5
    PRINT REG 0
    RET
