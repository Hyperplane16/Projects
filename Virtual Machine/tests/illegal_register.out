Register 5 cannot be accessed by instructions.
1